### PR TITLE
dep updates

### PR DESCRIPTION
**Pull request by jaredperreault-okta**

### CODE REVIEW

File: custom-login/package.json [hunk 1]
```diff
@@ -13,16 +13,15 @@
     "postinstall": "npm run patch-semantic-ui-css --prefix ../"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "^7.8.1",
-    "@okta/okta-signin-widget": "^7.24.2",
+    "@okta/okta-auth-js": "~7.9.0",
+    "@okta/okta-signin-widget": "~7.26.1",
```
Comment (lesterchoi-okta) [inline custom-login/package.json:17]: Why the change from compatible to approximate version range? IIUC this will prevent update to `@okta/okta-auth-js@7.10.0` and `@okta/okta-signin-widget@7.27.0` respectively

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies
https://docs.npmjs.com/cli/v6/using-npm/semver#tilde-ranges-123-12-1

File: package.json [hunk 2]
```diff
@@ -35,21 +34,20 @@
   "bugs": "https://github.com/okta/samples-js-vue/issues",
   "homepage": "https://github.com/okta/samples-js-vue",
   "devDependencies": {
-    "axios": "^1.6.4",
-    "dotenv": "^8.6.0",
+    "axios": "^1.7.9",
+    "dotenv": "^16.4.7",
```
Comment (lesterchoi-okta) [inline package.json:38]: :whistle: that's a big bump. dotenv's surface area is relatively small, but you've checked this doesn't break?

File: package.json [hunk 3]
```diff
@@ -35,21 +34,20 @@
...
+  },
+  "overrides": {
+    "semver": "6.3.1"
   },
   "bin": {},
-  "private": true,
-  "dependencies": {
-    "express": "^4.18.2"
-  }
+  "private": true
```
[the remark this answers is not in the document]
Comment (jaredperreault-okta) [reply package.json:52]: moved to devDeps and update from 4.18 to 4.21

**Review by lesterchoi-okta — APPROVED** [review]

**Review by denysoblohin-okta — APPROVED** [review]
